src/listen.js

```diff
@@ -152,6 +152,23 @@ module.exports = function(defaultFuncs, api, ctx) {
                 })(0)
                 break;
               }
+              
+              if (v.delta.class == "ClientPayload") {
+                var clientPayload = utils.decodeClientPayload(v.delta.payload);
+                if (clientPayload && clientPayload.deltas) {
+                  for (var i in clientPayload.deltas) {
+                    var delta = clientPayload.deltas[i];
+                    if (delta.deltaMessageReaction) {
+                      delta.deltaMessageReaction.type = "message_reaction";
+                      delta.threadId = delta.threadKey.threadFbId;
```

Comment:
Strange, after decoding your clientPayload, I got this response:
![image](https://cloud.githubusercontent.com/assets/1665373/24498397/a4c3dc10-153e-11e7-8e31-a7329d857999.png)
It's different than mine.